content/en/developers/guide/unified-tagging-advanced-usage.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Unified Tagging Advanced Usage Guide
+kind: guide
+aliases:
+  - /developers/guide/unified-tagging-advanced-usage
+---
+
+## Overview
+
+This guide suggests ways to configure and migrate to [unified service tagging][1] based on specific use cases.
+
+## Using Custom Tags
+
+You can continue to use the `env` `service` and `version` tags as they are configured for unified service tagging. However, if you'd like to achieve a unified tagging experience with your own custom tags, there are some options available to you:
+
+**Note**: While some products support arbitrary tags, others have more specific expectations. As such, navigation between products might be difficult if one data source has a tag that another does not have or does not support.
```

Comment:
Drop the `Note` per Yair's feedback on the other PR?

Reply:
His note was pertaining to having a note at the start of a section right below the header, which makes sense. It's a little odd to start a section with a note. However, I think this note make sense here as it is something we want people to make note of before going through these options.